### DISCUSSION TITLE
Which AWS Database service and postgres engine version is trustable to be used in production ?

### DISCUSSION THREAD

**Discussion by alirezaeeeali** [keycloak/keycloak#19653] [Q&A]
I want to deploy a self-hosted AWS EKS keycloak cluster with AWS-managed database service with scalability features. From the documentations I just saw that Postgres version 10 is compatible with keycloak, but the issue is AWS Aurora is not having that version available.
Also, another question that I have is whether it is a good idea to use a managed service in this manner or should I self-host the database as well?

**Comment by kwame-mintah** [comment]
@alirezaeeeali Have used Amazon Aurora with Keycloak in the past, the `Engine version` used at the time was 13.8. Before moving to Aurora, Keycloak used Amazon RDS.
